library/src/hcc_detail/hipsolver.cpp

```diff
@@ -396,6 +397,8 @@ hipsolverStatus_t hipsolverSorgbr_bufferSize(hipsolverHandle_t   handle,
                                              int*                lwork)
 try
 {
+    if(!handle)
+        return HIPSOLVER_STATUS_NOT_INITIALIZED;
```

Comment:
This is only covering the very specific case where both handle and lwork are null, and making it return HIPSOLVER_STATUS_NOT_INITIALIZED rather than HIPSOLVER_STATUS_INVALID_VALUE. A null handle should otherwise be detected when we call the rocSOLVER function in the memory size query.

Unfortunately, the bufferSize methods are not tested right now, and even if they were, the current bad arg test wouldn't have caught this. In the bad arg tests we have, the null handle test only sets the handle to be null.

Reply:
I can live with that. However, this is a user-visible change, so it deserves a mention in the changelog.